draft-ietf-scitt-architecture.md

```diff
@@ -437,7 +436,7 @@ In these cases, distinct Issuers (`iss`) might agree to use the `sub` CWT Claim
 
 Transparency Service:
 
-: an entity that maintains and extends the Append-only Log, and endorses its state.
+: an entity that maintains and extends the Verifiable Data Structure, and endorses its state.
 A Transparency Service can be a complex system, requiring the Transparency Service to provide many security guarantees about its Append-only Log.
```

Comment:
```suggestion
```

I was going to do edit this sentence, but I think it's actually almost information-free, and so I propose we drop it.